csrc/ir/utils.cpp

```diff
@@ -417,6 +412,21 @@ std::vector<TensorView*> allTvs(Fusion* fusion) {
   return uniqueEntries<TensorView>(all_tvs);
 }
 
+VectorOfUniqueEntries<TensorView*> allTvsOfExprs(
+    const std::vector<Expr*>& exprs) {
+  VectorOfUniqueEntries<TensorView*> all_tvs;
+  for (auto expr : exprs) {
+    auto input_tvs = ir_utils::filterByType<TensorView>(expr->inputs());
+    auto output_tvs = ir_utils::filterByType<TensorView>(expr->outputs());
+    for (const auto& tvs : {input_tvs, output_tvs}) {
+      for (auto tv : tvs) {
+        all_tvs.pushBack(tv);
+      }
+    }
```

Comment:
```suggestion
    auto append_all_to = [](const std::vector<TensorView*>& tvs, VectorOfUniqueEntires<TensorView*>& destination) {
      destination.pushBack(tvs.begin(), tvs.end());
    };
    append_all_to(ir_utils::filterByType<TensorView>(expr->inputs()), all_tvs);
    append_all_to(ir_utils::filterByType<TensorView>(expr->outputs()), all_tvs);
```

Reply:
`filterByType` does not create a vector. The current code seems just fine to me.